pages/k8s/install-local.md

```diff
@@ -135,11 +135,24 @@ All that remains is to deploy **Charmed Kubernetes**. A simple install can be ac
 juju deploy charmed-kubernetes
 ```
 
+
 This will install the latest stable version of **Charmed Kubernetes** with
 the default components and configuration. If you wish to customise this install
 (which may be helpful if you are close to the system requirements), please see
 the [main install page][install].
 
+<div class="p-notification--warning"><p markdown="1" class="p-notification__response">
+<span class="p-notification__status">Bug Warning:</span>
+There is currently a bug, <a href="https://bugs.launchpad.net/charm-kubernetes-worker/+bug/1903566"> LP#1903566</a>,
+which prevents <code>Kubelet</code> from running properly on LXD. Until this is fixed, a 
+workaround is to configure kubelet to protect the kernel defaults:
+  
+<pre><code>
+juju config kubernetes-worker kubelet-extra-config='{protectKernelDefaults: false}'
+</code></pre>
```

Comment:
The `<pre>` and `</pre>` seem to be showing up as text on the preview site. Not sure what's going on there.

https://deploy-preview-557--cdk-docs-next.netlify.app/kubernetes/docs/install-local

Reply:
yeah, I know, trying to fix that

Reply:
![Screenshot from 2021-04-23 16-18-36](https://user-images.githubusercontent.com/115290/115893013-a276d300-a44f-11eb-960d-cbe122684eda.png)

no idea why `<pre>`  doesn't work, but this looks right now at least